### ISSUE TITLE
xsd:hexBinary typed literals appear to be stored as actual strings

### ISSUE THREAD

**Issue by ebremer** [rdfhdt/hdt-java#123]
xsd:hexBinary typed literals appear to be stored as an actual string rather than converted to binary.  This seems to be an opportunity to get further compression.

**Comment by ebremer** [comment]
Same seems to be true for xsd:base64Binary

**Comment by mielvds** [comment]
Fair point! The gain would be minimal though, unless you have a bunch of those. Would you mind submitting a PR?

**Comment by ebremer** [comment]
I would have a quite a bunch.  Does HDT's design allow for (if coded up) storage of the hexbins and base64's as actually binary data?  If so, I would pursue it if it would be a welcome modification.

**Comment by mielvds** [comment]
It would! But it would also introduce a breaking change in the HDT files, meaning we'd have to move up a version and also reflect the changes in the C++ version. Not sure if that's worth it ATM without other improvements to release. That said, a branch is always welcome, ready to be included in a next release.
